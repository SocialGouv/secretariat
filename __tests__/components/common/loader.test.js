import { render } from "@testing-library/react"

import Loader from "../../../src/components/common/loader"

it("renders small loader", () => {
  const { container } = render(<Loader size="sm" />)
  expect(container).toMatchSnapshot()
})

it("renders medium loader", () => {
  const { container } = render(<Loader />)
  expect(container).toMatchSnapshot()
})

it("renders large loader", () => {
  const { container } = render(<Loader size="large" />)
  expect(container).toMatchSnapshot()
})
