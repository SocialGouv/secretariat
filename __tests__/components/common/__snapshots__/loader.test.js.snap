// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders large loader 1`] = `
<div>
  <div
    class="loader large"
  >
    <div />
  </div>
</div>
`;

exports[`renders medium loader 1`] = `
<div>
  <div
    class="loader md"
  >
    <div />
  </div>
</div>
`;

exports[`renders small loader 1`] = `
<div>
  <div
    class="loader sm"
  >
    <div />
  </div>
</div>
`;
